src/components/comment/comment.scss

```diff
@@ -83,6 +83,23 @@
   margin-top: math.div(size.$rhythm-condensed, -4);
 }
 
+// We added default styles to the blockquote element using the `wp-block-quote`
+// class. We're applying those same styles here, so comment blockquotes look
+// the similar.
+
+.c-comment__content blockquote {
+  border-left: size.$edge-medium solid color.$base-gray-light;
+  color: color.$base-gray-dark;
+}
+
+.c-comment__content blockquote cite {
+  font-style: normal;
+}
+
+.c-comment__content blockquote cite::before {
+  content: '\2014';
+}
```

Comment:
Since these styles are repeated here and in our WordPress vendor styles, should we create a Sass mixin and/or design tokens for them?

Reply:
That's a good point, I think tokens would work well here.

Reply:
@tylersticka I added tokens for most of these styles. However, I'm having trouble using a token for `font-style: normal`. I added a token in `tokens/font/styles.json`, but I get an error when I try to use it.